tools/antithesis-go-instrumentor/cmd/cmd_files.go

```diff
@@ -258,6 +286,10 @@ func (cfx *CommandFiles) FindSourceCode() (paths []string, numSkipped int, err e
 				return nil
 			}
 			if !strings.HasSuffix(path, ".go") {
+				if strings.HasSuffix(path, "go.mod") {
```

Comment:
https://pkg.go.dev/path#Base

should probably be filepath.Base(path)=="go.mod"

Reply:
Done - uses `filepath.Split()` to get both `dir` and ` baseFile`, then compares `baseFile` to "go.mod" and if they are equivalent, will consider `dir` as containing a submodule.

Reply:
Done